core/domain/user_services.py

```diff
@@ -43,7 +43,7 @@
 
 # Size (in px) of the gravatar being retrieved.
 GRAVATAR_SIZE_PX = 150
-# Data url for images/avatar/user_blue_72px.png.
+# Data url for images/avatar/user_blue_72px.webp.
```

Comment:
This is not true right? I still see `data:image/png` below

Reply:
> @darksun27, Thanks for teh PR I looked at codeowner files (And some important files) and left a few comments, PTAL!
> 
> Also, I have a few questions in general:
> 
>     1. Any specific reason for the changes in `assets/images/icons/rocket@2x.png`?
>        ![image](https://user-images.githubusercontent.com/16653571/83948786-69387300-a83d-11ea-85ad-5b9a606049db.png)
No specific reasons as to why there is a decrease in the size. I'll just swap this image to the one in the codebase.
> 
>     2. Do we need to delete the .png files which are not user anywhere?
Most of the png files are being used. I apparently didn't see any png file not used.

Reply:
> This is not true right? I still see `data:image/png` below

done!